spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/test/java/org/springframework/cloud/gcp/sql/autoconfig/GcpCloudSqlAutoConfigurationMockTests.java

```diff
@@ -80,14 +82,14 @@ public void test() {
 					+ "&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false",
 					this.urlProvider.getJdbcUrl());
 			assertEquals("root", dataSource.getUsername());
-			assertEquals("", dataSource.getPassword());
+			assertEquals(null, dataSource.getPassword());
```

Comment:
This should be an empty string by default.

Reply:
I think we need it to be an empty string, otherwise it won't work with the default configuration of Cloud SQL instance ("root", "").